src/main/resources/application.yml

```diff
@@ -63,45 +63,23 @@ spring:
     name: ONS RespondentHomeService
   lifecycle:
     timeout-per-shutdown-phase: 30s
-  rabbitmq:
-    username: guest
-    password: guest
-    host: localhost
-    port: 35672
-    virtualhost: /
+  cloud:
+    gcp:
+      pubsub:
+        project-id: local
+        emulator-host: localhost:8085
 
 queueconfig:
-  event-exchange: events
-  dead-letter-exchange: events.deadletter.exchange
-  case-queue: case.rh.case
-  case-queue-DLQ: case.rh.case.dlq
-  case-routing-key: event.case.update
-  uac-queue: case.rh.uac
-  uac-queue-DLQ: case.rh.uac.dlq
-  uac-routing-key: event.uac.update
-  response-authentication-routing-key: event.response.authentication
+  case-subscription: case-rh-case
```

Comment:
if we have a choice I would like to drop the "case-" prefix so that the subscriptions become
- rh-case
- rh-uac